eZ Platform/Admin UI/REQUIREMENTS.md

```diff
@@ -0,0 +1,45 @@
+# v2 Hybrid Prototype Admin UI Requirements
+## https://github.com/ezsystems/specs/issues/4
+
+# Objective
+"As an eZ Engineer, I want...
```

Comment:
Good that we have some clarifications on what we want to archive and for who we prototype, but I'm missing the root objectives/user stories that solve Developer persona issues here, to not lose track of what we in the end try to solve.